File: app/src/main/java/starbright/com/projectegg/data/local/model/Instruction.java
package starbright.com.projectegg.data.local.model;

public class Instruction {

    private String mNumber;
    private String mStep;

    public Instruction(String number, String step) {
        mNumber = number;
        mStep = step;
    }

    public String getNumber() {
        return mNumber;
    }

    public void setNumber(String number) {
        mNumber = number;
    }

    public String getStep() {
        return mStep;
    }

    public void setStep(String step) {
        mStep = step;
    }
}

File: app/src/main/java/starbright/com/projectegg/data/model/Instruction.kt
/*
 * Copyright (c) by Andreas (oentoro.andreas@gmail.com)
 * created at 25 - 7 - 2020.
 */

/**
 * Created by Andreas on 15/8/2018.
 */
package starbright.com.projectegg.data.model

data class Instruction(
    var name: String = "",
    var steps: List<String> = emptyList()
) {
    // Keep backward compatibility constructor
    constructor(number: Int, step: String) : this(
        name = "Step $number",
        steps = listOf(step)
    )
}
